src/nwb2bids/_tools/_cache_nwb.py

```diff
@@ -0,0 +1,18 @@
+import functools
+import pathlib
+import warnings
+
+import pynwb
+
+warnings.filterwarnings(action="ignore", message="No cached namespaces found in .*")
```

Comment:
```suggestion
warnings.filterwarnings(action="ignore", message="No cached namespaces found in .+")
```

The current one is really no different than just

```py
warnings.filterwarnings(action="ignore", message="No cached namespaces found in ")
```

I suggest the same change to the other lines of `warnings.filterwarnings` as well.

I think I am nitpicking here, so ignore the suggestion if you choose.